src/hooks/usePreloadLazyModules.ts

```diff
@@ -0,0 +1,8 @@
+// This hook makes it possible to manually load the lazy loaded modules
+// right after the main bundle is loaded to decrease the bundle size
+function usePreloadLazyModules() {
+    import(/* webpackPrefetch: true */ 'react-fast-pdf');
+    import(/* webpackPrefetch: true */ 'react-pdf');
+}
```

Comment:
Suggestion for consideration:
You can do some additional optimization and trigger additional JS loading when the browser is idle.

https://developer.mozilla.org/en-US/docs/Web/API/Window/requestIdleCallback

```js
requestIdleCallback(callback)
requestIdleCallback(callback, options)
```